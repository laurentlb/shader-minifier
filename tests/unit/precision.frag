#version 300 es

precision highp float;
precision mediump int;
precision lowp sampler2D;
precision lowp samplerCube;

lowp float foo(lowp float bar) {
  return 1.0;
}
