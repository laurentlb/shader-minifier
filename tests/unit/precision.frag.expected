#version 300 es

precision highp float;
precision mediump int;
precision lowp sampler2D;
precision lowp samplerCube;
lowp float f(lowp float f)
{
  return 1.;
}
